src/jit/emitxarch.cpp

```diff
@@ -7026,7 +7026,17 @@ void                emitter::emitDispIns(instrDesc*   id,
         {
             printf("%s, %s", emitRegName(id->idReg2(), attr), emitXMMregName((unsigned)id->idReg1()));
         }
-        else if  (ins == INS_cvttsd2si)
+#ifdef _TARGET_AMD64_
```

Comment:
cvt* are not amd64 specific instructions. They belong to x86 as well.
Looking at instrxarch.h, these are under ```#ifndef LEGACY_BACKEND```. So should we do the same thing here?